CHANGELOG.md

```diff
@@ -29,6 +31,16 @@ instructions](https://nasa.github.io/cumulus/docs/upgrade-notes/upgrade-rds-clus
   - Update `cumulus-rds-tf` module to take additional parameters in support of
     migration from Aurora PostgreSQl v11 to v13.   See Migration Notes for more details.
 
+## [v16.1.4] 2024-2-16
+
+Please note changes in 16.1.4 may not yet be released in future versions, as this
+is a backport/patch release on the 16.x series of releases. Updates that are
+included in the future will have a corresponding CHANGELOG entry in future releases.
+
+### Changed 
+- **CUMULUS_3547
```

Comment:
```suggestion
- **CUMULUS_3547**
```

Reply:
@kwartrosa reminder to respond to comments with a commit or explanation. If the author 'resolves' the comments, the reviewer must dig to figure out if they've been addressed.

Reply:
'**CUMULUS_3547**' accepted commit suggestion to address formatting issue.